components/t547/display.py

```diff
@@ -23,6 +23,7 @@
         {
             cv.GenerateID(): cv.declare_id(T547),
             cv.Optional(CONF_GREYSCALE, default=False): cv.boolean,
+            cv.Optional(CONF_WHITE_IS_ON, default=False): cv.boolean,
```

Comment:
I've set the default to False to not break existing code/implementations/projects.
Maybe it would be better/nicer to make the default depend on Greyscale?